src/spec/intersection_spec.js

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai'
+import intersection from '../lib/intersection.js'
+const expect = chai.expect
+
+describe('Intersection', () => {
+  it('Creates an array of unique values that are included in all given arrays', () => {
+    expect(intersection([4,2,7,5],[4,3,8,6])).to.eql([4])
+  })
+  it('Creates an array of unique values that are included in all given arrays', () => {
```

Comment:
this test can either be dropped or merged into the previous one-- it is testing for the same thing with a different set of arrays.